examples/README.md

```diff
@@ -2695,8 +2692,7 @@ If the retries are not successful, we want to just gracefully end workflow execu
       ],
       "onErrors": [
         {
-          "error": "ServiceNotAvailable",
-          "code": "503",
+          "condition": "${ .error.message == 'ServiceNotAvailable' and .error.code == 503 }",
```

Comment:
should return true/false imo here too..so in this case use

any(.exceptions[]; .error == "ServiceNotAvailable" and .code == "503")